Tests/js/views/services/plugins/assets/ez-searchplugin-tests.js

```diff
@@ -111,7 +147,8 @@ YUI.add('ez-searchplugin-tests', function (Y) {
             this.view.fire('contentSearch', {
                 viewName: this.viewName,
                 search: {
-                    criteria: {},
+                    offset: 42,
```

Comment:
then it's completely useless and you fix your mock instead of changing each and every test